### DISCUSSION TITLE
无法修改书签名称

### DISCUSSION THREAD

**Discussion by AvrilLavigneDan** [electerm/electerm#3400] [Q&A]
![1714961762533](https://github.com/electerm/electerm/assets/151896553/d068e86a-2ef8-4578-bb5f-6b880b1a49c5)
我想修改本地书签的名称，没有找到修改的按钮

**Comment by zxdong262** [comment]
请问是最新版吗

**Comment by AvrilLavigneDan** [comment]
1.38.86版本

**Comment by zxdong262** [comment]
![Screen Shot 2024-05-06 at 14 20 08](https://github.com/electerm/electerm/assets/1641949/ee44d4cc-f3c2-4619-8447-831b73e64fed)
有这个按钮吗

**Comment by AvrilLavigneDan** [comment]
找到了，感谢
